internal/server/task.go

```diff
@@ -120,6 +123,14 @@ func Task(client TaskClient, tmpl template.Template) Handler {
 				return err
 			} else {
 				data.Success = true
+
+				SetFlash(w, FlashNotification{
+					Title: "Task created",
+				})
+
+				if data.Case.CaseType == "DIGITAL_LPA" {
+					return RedirectError(fmt.Sprintf("/lpa/%s", data.Case.UID))
+				}
```

Comment:
We should only set the flash notification if we know we're going to show it immediately afterwards. otherwise it will show up on whatever the next digital LPA they open is, which would be confusing.
```suggestion
				if data.Case.CaseType == "DIGITAL_LPA" {
					SetFlash(w, FlashNotification{
						Title: "Task created",
					})

					return RedirectError(fmt.Sprintf("/lpa/%s", data.Case.UID))
				}
```